src/results/ashp.jl

```diff
@@ -0,0 +1,126 @@
+# REopt®, Copyright (c) Alliance for Sustainable Energy, LLC. See also https://github.com/NREL/REopt.jl/blob/master/LICENSE.
+
+"""
+`ASHP` results keys:
+- `size_ton`  # Thermal production capacity size of the ASHP [ton/hr]
+- `electric_consumption_series_kw`  # Fuel consumption series [kW]
+- `annual_electric_consumption_kwh`  # Fuel consumed in a year [kWh]
+- `thermal_production_series_mmbtu_per_hour`  # Thermal energy production series [MMBtu/hr]
+- `annual_thermal_production_mmbtu`  # Thermal energy produced in a year [MMBtu]
```

Comment:
Let's add in the comment string that this _mmbtu is **heating** while the _tonhour is **cooling** since we don't distinguish between heating and cooling thermal production in the name.

Reply:
Updated here https://github.com/NREL/REopt.jl/pull/372/commits/ab4b107873012068db45e6906a5d30eb094819cc